### PR TITLE
Create makefile for compression

### PR DESCRIPTION
**Pull request by tallowen**
Reorganized directory structure to better make clear what is mine, what
is for the demo and what gets compiled

### CODE REVIEW

**Comment by tofumatt** [comment]
That's a good idea!

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+LESS_COMPRESSOR ?= `which lessc`
+RESOURCE_FOLER = ./media/
```
Comment (tofumatt) [inline Makefile]: `RESOURCE_FOLDER` would make more sense.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+compile:
+	mkdir -p ${RESOURCE_FOLER}
+	lessc src/style.less --yui-compress >> ${RESOURCE_FOLER}socialshare.min.css
```
Comment (fwenzel) [inline Makefile]: I think we might need to tell people in the readme what lessc is. But you were going to have to add some more stuff to the readme anyway, so.

**Comment by fwenzel** [comment]
Fine fine, just make it a shell script then.

**Pull request merged**